src/Microsoft.DotNet.Wpf/src/WpfGfx/core/common/renderoptions.cpp

```diff
@@ -23,10 +23,17 @@ RenderOptions_IsSoftwareRenderingForcedForProcess()
     return RenderOptions::IsSoftwareRenderingForcedForProcess();
 }
 
+void WINAPI
+RenderOptions_SetGraphicHWAccelerationForRdp(BOOL value)
```

Comment:
More descriptive API names would be helpful. `BOOL enable` or `BOOL fEnable`. And perhaps the method name too: `RenderOptions_EnableGraphicsHardwareAccelerationForRdp`

Reply:
thank you for your feedback. I think this make sense.